README.md

```diff
@@ -11,64 +11,51 @@ Uses request subdomain to swap database, Rails cache (preferences), image paths.
 
 ### Installation
 
-Add to your `Gemfile`:
+1. Add to your `Gemfile`:
 
-```ruby
-gem 'spree_shared', github: 'spree-contrib/spree_shared', branch: 'master'
-```
+    ```ruby
+    gem 'spree_shared', github: 'spree-contrib/spree_shared', branch: 'master'
+    ```
 
-Make sure your `config/database.yml` has valid db connection.
+    Make sure your `config/database.yml` has valid db connection.
 
-Create an initializer `config/initializers/apartment.rb` with the following command:
+2. Create `config/initializers/apartment.rb`with the following command:
 
-```bash
-bundle exec rails generate apartment:install
-```
+    ```bash
+    bundle exec rails generate apartment:install
+    ```
 
-Search for following line:
+3. Search for following line inside `config/initializers/apartment.rb`:
 
-```ruby
-Apartment.configure do |config|
-  ...
-  # supply list of database names for migrations to run on
-  config.tenant_names = lambda { ToDo_Tenant_Or_User_Model.pluck :database }
-end
-```
+    ```ruby
+      config.tenant_names = lambda { ToDo_Tenant_Or_User_Model.pluck :database }
```

Comment:
That's because its a part of enumerated point in the points list. See the rich diff:
![Screenshot 2020-11-25 at 14 42 34](https://user-images.githubusercontent.com/9699466/100235533-d5627c80-2f2c-11eb-8d32-010d1ce6f2a9.png)